### PR TITLE
Adding several testing tools

### PR DESCRIPTION
**Pull request by Grummfy**

### CODE REVIEW

File: _tools/behat.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+authors:        [Konstantin Kudryashov]
+license:        {url: 'https://github.com/Behat/Behat/blob/master/LICENSE', label: 'MIT'} 
+docs:           {url: 'http://behat.org/en/latest/guides.html'} 
+github:         {name: 'Behat/Behat'} 
+packagist:      {name: 'behat/behat'}
+command:        behat        
+tags:           [testing, bdd]
+---
+
+[{{ page.title }}]({{ site.url }}{{ page.url }}) Behat is an open source [Behavior-Driven Development](https://en.wikipedia.org/wiki/Behavior-driven_development) (BDD) framework for PHP.
```
Comment (MetalArend) [inline _tools/behat.md]: Behat is shown twice here.

File: _tools/kahlan.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+layout:         tool
+title:          Kahlan
+website:        {url: 'https://kahlan.github.io/docs/'} 
+authors:        [CrysaLEAD]
+license:        {url: 'https://github.com/kahlan/kahlan/blob/master/LICENSE.txt', label: 'MIT'} 
+docs:           {url: 'https://kahlan.github.io/docs/'} 
+github:         {name: 'kahlan/kahlan'} 
+packagist:      {name: 'kahlan/kahlan'}
+dockerhub:      ({name: 'kahlan/kahlan'})
```
Comment (MetalArend) [inline _tools/kahlan.md]: This should be an object, the "()" is too much.

Comment (Grummfy) [inline _tools/kahlan.md]: it's inside the draft ;)

File: _tools/phpspec.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+packagist:      {name: 'phpspec/phpspec'}
+command:        phpspec
+dependencies:   [prophecy]
+tags:           [testing, specification]
+---
+
+Spec BDD with phpspec. phpspec is a tool which can help you write clean and working PHP code using behaviour
+driven development or BDD. BDD is a technique derived from test-first development. 
+
+[{{ page.title }}]({{ site.url }}{{ page.url }}) ...
```
Comment (MetalArend) [inline _tools/phpspec.md]: Can you check this paragraph?

File: _tools/simpletest.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+title:          SimpleTest
+website:        {url: 'http://simpletest.org/'} 
+authors:        [Marcus Baker, Jason Sweat, Travis Swicegood, Perrick Penet, Edward Z. Yang, Jens A. Koch]
+license:        {url: 'https://github.com/simpletest/simpletest/blob/master/LICENSE', label: 'LGPL-2.1'} 
+github:         {name: 'simpletest/simpletest'} 
+packagist:      {name: 'simpletest/simpletest'}
+tags:           [testing]
+---
+
+[{{ page.title }}]({{ site.url }}{{ page.url }}) a unit test framework for PHP.
```
Comment (MetalArend) [inline _tools/simpletest.md]: Can you add an "is" here?

**Comment by MetalArend** [comment]
Wow, that's eight more tools.

**Comment by Grummfy** [comment]
it should be ok now ;)

**Comment by MetalArend** [comment]
All added, thanks!

**Pull request merged**